hazelcast/src/main/java/com/hazelcast/client/impl/spi/ProxyManager.java

```diff
@@ -154,6 +193,20 @@ public void init(ClientConfig config, ClientContext clientContext) {
         }
 
         readProxyDescriptors();
+        executor.schedule(new SyncDistributedObjectsTask(),
+                DISTRIBUTED_OBJECT_SYNC_PERIOD_MILLIS, TimeUnit.MILLISECONDS);
+    }
+
+    private ScheduledExecutorService createExecutorService() {
+        ClassLoader classLoader = client.getClientConfig().getClassLoader();
+        ILogger logger = client.getLoggingService().getLogger(ProxyManager.class);
+        String name = client.getName();
+        return new LoggingScheduledExecutor(logger, EXECUTOR_POOL_SIZE,
+                new PoolExecutorThreadFactory(name + ".internal-", classLoader), (r, executor) -> {
```

Comment:
Lets make the name postfix as `.proxy.internal-` to differentiate this from the others.